examples/motion_viewer.py

```diff
@@ -76,6 +105,7 @@ def play_motion() -> None:
             if self.fm_demo.motion is not None:
                 self.fm_demo.next_pose()
                 self.fm_demo.next_pose()
```

Comment:
Why is this called twice? If it is to sync the 120Hz motion with the 60Hz viewer update then I suggest a slightly more general formulation (e.g. pass the timestep into next_pose and handle frame choice and tracking internally instead of assuming a hardcoded 2/1 ratio.